pxb-tag/package.json

```diff
@@ -32,7 +32,7 @@
     ],
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 0",
-        "tag-package": "npx -p @pxblue/tag pxb-tag",
-        "publish:package": "set npm_config_yes=true && npx -p @pxblue/publish pxb-publish"
+        "publish:package": "set npm_config_yes=true && npx -p @pxblue/publish pxb-publish",
+        "tag:package": "npx -p @pxblue/tag pxb-tag"
```

Comment:
this one should be the local bash file:
![image](https://user-images.githubusercontent.com/29152776/126380591-69bcbf77-81bf-4e48-98e7-85cd1cb98a1d.png)

Reply:
It is using the local code; the tag step can only be ran after the publish step is ran.  Since tag only runs on master branch, I can't see a scenario where it doesn't use the latest from NPM.

Reply:
Where NPM === local

Reply:
The only reason why I don't want to run the file like `sh scripts/tag.sh` is because the `pxb-parse-changelog` command inside the tag script will most-likely fail.  Downloading the same script and executing it through NPX will fetch the local code (which is now the latest publish) and will run successfully.

Reply:
Okay, fair point about it being published before the tag happens. Works for me.